client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -70,7 +70,7 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 	};
 
 	checkForEmptyFields = () => {
```

Comment:
While here, would you like to rename this one to something like `hasEmptyFields`?

Reply:
adapted